configuration/concepts/concepts.csv

```diff
@@ -0,0 +1,338 @@
+,,,Cambodia_Teeth Filled,Cambodia_Teeth Filled,Teeth Filled,Teeth Filled,,,Question,Coded,,CAMBODIA_Tooth 11; CAMBODIA_Tooth 12; CAMBODIA_Tooth 13; CAMBODIA_Tooth 14; CAMBODIA_Tooth 15; CAMBODIA_Tooth 16; CAMBODIA_Tooth 17; CAMBODIA_Tooth 18; CAMBODIA_Tooth 21; CAMBODIA_Tooth 22; CAMBODIA_Tooth 23; CAMBODIA_Tooth 24; CAMBODIA_Tooth 25; CAMBODIA_Tooth 26; CAMBODIA_Tooth 27; CAMBODIA_Tooth 28; CAMBODIA_Tooth 31; CAMBODIA_Tooth 32; CAMBODIA_Tooth 33; CAMBODIA_Tooth 34; CAMBODIA_Tooth 35; CAMBODIA_Tooth 36; CAMBODIA_Tooth 37; CAMBODIA_Tooth 38; CAMBODIA_Tooth 41; CAMBODIA_Tooth 42; CAMBODIA_Tooth 43; CAMBODIA_Tooth 44; CAMBODIA_Tooth 45; CAMBODIA_Tooth 46; CAMBODIA_Tooth 47; CAMBODIA_Tooth 48; CAMBODIA_Tooth 51; CAMBODIA_Tooth 52; CAMBODIA_Tooth 53; CAMBODIA_Tooth 54; CAMBODIA_Tooth 55; CAMBODIA_Tooth 61; CAMBODIA_Tooth 62; CAMBODIA_Tooth 63; CAMBODIA_Tooth 64; CAMBODIA_Tooth 65; CAMBODIA_Tooth 71; CAMBODIA_Tooth 72; CAMBODIA_Tooth 73; CAMBODIA_Tooth 74; CAMBODIA_Tooth 75; CAMBODIA_Tooth 81; CAMBODIA_Tooth 82; CAMBODIA_Tooth 83; CAMBODIA_Tooth 84; CAMBODIA_Tooth 85;,,,,,,,,,
+,,,Cambodia_Is Small Dental Surgery Performed,Cambodia_Is Small Dental Surgery Performed,Small Surgery Performed,Small Surgery Performed,,,Question,Boolean,,,,,,,,,,,
+,,,Cambodia_Stitches (Suture),Cambodia_Stitches (Suture),Stitches (Suture),Stitches (Suture),,,Misc,N/A,,,,,,,,,,,
+,,,Cambodia_Drain Dental Abscess,Cambodia_Drain Dental Abscess,Drain Dental Abscess,Drain Dental Abscess,,,Misc,N/A,,,,,,,,,,,
+,,,Cambodia_Coded Small Dental Surgery Performed,Cambodia_Coded Small Dental Surgery Performed,Small Surgery Type,Small Surgery Type,,,Question,Coded,,Cambodia_Stitches (Suture); Cambodia_Drain Dental Abscess,,,,,,,,,
+,,,Cambodia_Other Small Dental Surgery,Cambodia_Other Small Dental Surgery,Other Small Surgery,Other Small Surgery,,,Question,Text,,,,,,,,,,,
+,,,Cambodia_Small Dental Surgery Set,Cambodia_Small Dental Surgery Set,Small Surgery,Small Surgery,,,ConvSet,N/A,,,Cambodia_Coded Small Dental Surgery Performed; Cambodia_Other Small Dental Surgery,,,,,,,,
+,,,Cambodia_Scaling,Cambodia_Scaling,Scaling,Scaling,,,Question,Boolean,,,,,,,,,,,
+,,,Cambodia_Prevention Education,Cambodia_Prevention Education,Prevention Education,Prevention Education,,,Question,Boolean,,,,,,,,,,,
+,,,Cambodia_Prevention Fluoride,Cambodia_Prevention Fluoride,Fluoride Treatment,Fluoride Treatment,,,Question,Boolean,,,,,,,,,,,
+,,,Cambodia_Dental Notes,Cambodia_Dental Notes,Dental Notes,Dental Notes,,,Question,Text,,,,,,,,,,,
+,,,Cambodia_Dental Exam Form,Cambodia_Dental Exam Form,Dental Exam,Dental Exam,,,ConvSet,N/A,,,Cambodia_Missing Teeth; Cambodia_Teeth Extracted; Cambodia_Teeth Filled;Cambodia_Small Dental Surgery Set; Cambodia_Scaling; Cambodia_Prevention Education; Cambodia_Prevention Fluoride;Cambodia_Dental Notes,,,,,,,,

```

Comment:
Could you extract these concepts to be in a seperate CSV. Basically the idea is to have 1 form per CSV.

Reply:
After discussing with @rubailly , we'll do this through another ticket/PR